vendor/github.com/stretchr/testify/mock/mock.go

```diff
@@ -18,6 +18,9 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// regex for GCCGO functions
+var gccgoRE = regexp.MustCompile(`\.pN\d+_`)
```

Comment:
not ideal in runtime code but should be fine in a mock package.

Reply:
I guess we don't have any way to ensure it's test-import only? something to investigate in the future.